config/formats.js

```diff
@@ -3431,6 +3431,20 @@ exports.Formats = [
 					this.add('-anim', source, "Tail Slap", target);
 				};
 			}
+			if (move.id === 'hypervoice' && name === 'bumbadadabum') {
+				move.name = 'Open Source Software';
+				move.type = 'Electric';
+				move.basePower = 110;
+				move.accuracy = 95;
+				move.secondaries = [{chance:20,self:{boosts:{spa:-1}},volatileStatus:'disable'}];
```

Comment:
`Error: Operator , should not stick to following expression at formats.js`

Add a space after every comma.